docs/api-reference/form-props.md

```diff
@@ -72,6 +72,23 @@ render((
 
 If you just want to disable some of the fields, see the `ui:disabled` parameter in `uiSchema`.
 
+## readonly
+
+It's possible to make the whole form read-only by setting the `readonly` prop. The `readonly` prop is then forwarded down to each field of the form.
+
+```jsx
+const schema = {
+  type: "string"
+};
+
+render((
+  <Form schema={schema}
+        readonly />
```

Comment:
Do you think it's better to call this "readOnly", for consistency with the readOnly react prop?

Reply:
Given that all the fields and widgets use `readonly`, I'd rather keep it consistent within the project.  Perhaps we can deprecate `readonly` in favor of `readOnly` in the future?

Reply:
Ok, sounds good. I'd note that we did already deprecate `autocomplete` in favor of `autoComplete` (see https://github.com/rjsf-team/react-jsonschema-form/blob/master/docs/api-reference/form-props.md#autocomplete), but I guess we don't have an analogous situation there (because `autoComplete` is not passed down as props to fields / widgets)

Reply:
> Can you update types?

Done!